### PR TITLE
Add css lint support

### LINKED ISSUE #85: Lint CSS files?
Currently, our CSS files aren't very complex, but it may be worth considering adding something like [**stylelint**](http://npm.im/stylelint) to the tool belt.

Possible solution:

```js
// .stylelintrc
{
  "extends": "stylelint-config-standard",
  "ignoreFiles": [
    "public/css/vendor/*.css"
  ]
}
```

```diff
diff --git a/package.json b/package.json
index 9c75c03..0e4ff85 100644
--- a/package.json
+++ b/package.json
@@ -31,6 +31,8 @@
     "npm-run-all": "^4.1.2",
     "nsp": "^3.2.1",
     "nyc": "^11.5.0",
+    "stylelint": "^9.2.0",
+    "stylelint-config-standard": "^18.2.0",
     "tape": "^4.9.0"
   },
   "engines": {
@@ -49,6 +51,7 @@
   },
   "scripts": {
     "lint": "npm-run-all lint:*",
+    "lint:css": "stylelint 'public/css/**/*.css'",
     "lint:js": "eslint .",
     "lint:nsp": "nsp check",
     "pretest": "npm run lint",
```

That should lint all the CSS files (except the files in the [./public/css/vendor/](https://github.com/mozilla/blurts-server/tree/master/public/css/vendor) directory).

### PR DESCRIPTION
**Pull request by josechoy**
Fixes #85 

fixed `public/css/app.css` after `npm run lint`

### CODE REVIEW

**Comment by groovecoder** [comment]
@pdehaan do we use `stylelint` for CSS linting on other mozilla projects? Or is there some other linter you suggest?

**Comment by pdehaan** [comment]
@groovecoder: Yeah, **stylelint** is our go-to CSS linter in most projects. We've also used Sass Lint in a few projects, but it isn't great for vanilla CSS linting (whereas **stylelint** is customizable and works w/ Sass and other preprocessors).

**Comment by groovecoder** [comment]
Okay, this needs a conflict resolution in `app.css` ... @nhnt11 do you want to help @josechoy with that?

File: public/css/app.css [hunk 1]
```diff
@@ -149,47 +162,47 @@ img.logo {
...
 #breach-title-wrapper {
-  display:flex;
-  margin-bottom:calc(.5vh + .5vw);
+  display: flex;
+  margin-bottom: calc(0.5vh + 0.5vw);
 }
 
 #breach-title-date {
   display: flex;
-  align-items:center;
```
Comment (pdehaan) [inline public/css/app.css]: @lesleyjanenorton I wasn't sure what the right answer for this was. It looks like this selector had two `align-items` values ("center" and "flex-start"). I opted for the latter one, figuring it was probably overriding "center" anyways. But happy to change to a different value in this PR or a new one...

**Pull request merged**